docs/source/backends/adios2.rst

```diff
@@ -160,6 +160,23 @@ Ignore the 30GB initialization phases.
 .. image:: ./memory_groupbased_nosteps.png
   :alt: Memory usage of group-based iteration without using steps
 
+
+Known Issues
+------------
+
+.. warning::
+
+   Nov 1st, 2021 (`ADIOS2 2887 <https://github.com/ornladios/ADIOS2/issues/2887>`__):
+   libfabric 1.6+ introduced a couple of breaking changes that break ADIOS2 SST (staging/streaming) workflows.
+
+   The following environment variables can be set as work-arounds on Cray systems, when working with ADIOS2 SST:
+
+   .. code-block:: bash
+
+      export FABRIC_IFACE=mlx5_0   # ADIOS SST: select interface (1 NIC on Summit)
+      export FI_OFI_RXM_USE_SRX=1  # libfabric: use shared receive context from MSG provider
```

Comment:
Should we also add `FI_PROVIDER=verbs`? That's the fabric provider that ADIOS2 successfully selects with `libfabric1.6.2`  on Summit. 
I'm not really sure about the significance of `FI_PSM2_DISCONNECT=1` mentioned [here](https://github.com/ornladios/ADIOS2/issues/2887#issuecomment-939150608) and [here](https://github.com/ornladios/ADIOS2/issues/2887#issuecomment-945966477). I'm currently testing if that one makes a difference.

Reply:
Done. This being said, these variables did not give me successful runs at 1024 nodes (unlike I recently had with libfabric1.6), so I'll next need to have smaller-scale debug runs to see what's going on.